app/services/cache_manager.rb

```diff
@@ -10,7 +10,10 @@ def self.create_new_permission_cache
   end
 
   def self.create_new_filter_cache
-    Filter.all.each(&:save!)
+    Role.bypass_locking = true
+    Filter.all.map(&:save!)
+  ensure
```

Comment:
if we had a wrapping method, we wouldn't have to remember to do `ensure` whenever we bypass the locking